pkg/minishift/addon/addon_meta.go

```diff
@@ -134,6 +139,23 @@ func (meta *DefaultAddOnMeta) OpenShiftVersion() string {
 	return anyOpenShiftVersion
 }
 
+func (meta *DefaultAddOnMeta) Dependency() ([]string, error) {
+	if val, contains := meta.headers[dependsOn].(string); contains {
+		return minishiftStrings.SplitAndTrim(val, ",")
+	}
+	return []string{}, nil
+}
+
+func checkDependencySemantic(headers map[string]interface{}) bool {
```

Comment:
@gbraad The question i asked in IRC about the regex, here it is..  should i be using split here instead?